test/test_optimize_strategies.py

```diff
@@ -57,29 +58,31 @@ def params():
         ),
     ),
 )
-def test_index_search(caplog, responses, params, state, expires, expected, logs):
```

Comment:
This is fine, but fyi there exists an aioresponses mock library designed to be used with aiohttp that we could make use of in a similar way.